tags/feature_disabling/feature_disabling_tags.py

```diff
@@ -0,0 +1,4 @@
+from talon import Module
+
+mod = Module()
+mod.tag("screenshots_disabled", desc="Activating this tag disables screenshot commands")
```

Comment:
Everything in the plugin directory needs to be self contained. ie no dependencies on files outside of that directory. I would just move `mod.tag()` to `screenshot.py`

Reply:
Done. Thank you for the suggestion!